components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -476,6 +479,20 @@ private void validateAddOrganizationRequest(Organization organization) throws Or
         validateAddOrganizationType(organization);
     }
 
+    private void validateSameOrgNameExistOnImmediateSubOrganizationsOfParentOrg(Organization organization)
```

Comment:
What if we change this method signature to pass, organizationName, and the parent organization id.?

Reply:
Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/29/commits/2d2707a5a9061170fb00f35525013252425a8586